nexus/src/db/datastore.rs

```diff
@@ -2679,6 +2679,70 @@ impl DataStore {
             }),
         }
     }
+
+    // SSH public keys
+
+    pub async fn ssh_keys_list(
+        &self,
+        silo_user_id: Uuid,
+        page_params: &DataPageParams<'_, Name>,
+    ) -> ListResultVec<SshKey> {
+        use db::schema::ssh_key::dsl;
+
+        paginated(dsl::ssh_key, dsl::name, page_params)
+            .filter(dsl::silo_user_id.eq(silo_user_id))
+            .filter(dsl::time_deleted.is_null())
+            .select(SshKey::as_select())
+            .load_async(self.pool())
```

Comment:
use `pool_authorized(opctx).await?` here instead of `pool()`

Reply:
Fixed in 95114e05.